src/components/activity-page/embeddable.tsx

```diff
@@ -118,7 +118,14 @@ export const Embeddable: React.ForwardRefExoticComponent<IProps> = forwardRef((p
 
   const singlePageLayout = activityLayout === ActivityLayouts.SinglePage;
   const isFullWidthLayout = sectionLayout === "full-width" || singlePageLayout;
-  const singleWithMaxAspectRatio = isFullWidthLayout && (embeddable.type === "ManagedInteractive" ? embeddable.custom_aspect_ratio_method === "MAX" : embeddable.aspect_ratio_method === "MAX");
+
+  let hasMaxAspectRatio;
+  if (embeddable.type === "ManagedInteractive" && !embeddable.inherit_aspect_ratio_method) {
+    hasMaxAspectRatio = embeddable.custom_aspect_ratio_method === "MAX";
+  } else {
+    hasMaxAspectRatio = embeddable.aspect_ratio_method === "MAX";
+  }
+
```

Comment:
@lublagg I'm not sure but I think in this case the embeddable is the embeddable from the LARA JSON.

So if the `embeddable.type === "ManagedInteractive"` and `embeddable.inherit_aspect_ratio_method` is true, then the code should get the aspect ratio method from the library interactive by doing: `embeddable.library_interactive?.data.aspect_ratio_method`

In managed-interactive.tsx this is different because in that component, above the logic it is doing:
```
  if (embeddable.type === "ManagedInteractive") {
    embeddableData = embeddable.library_interactive?.data;
  } else {
    embeddableData = embeddable;
  }
```

Reply:
Hi Scott, I just pushed up a change to the existing logic that I think makes sense. 

hasMaxAspectRatio is false by default. The aspect_ratio_method on the data object of library_interactive is typed as only ever being able to be default or null, so I think it's safe to just check if the aspect ratio is "max" in the case that the managed interactive has a custom aspect ratio and is not supposed to inherit its aspect ratio, or if the type of the embeddable is MwEmbeddable.

Here is a screenshot of the approach I went with:

<img width="675" alt="Screen Shot 2022-10-04 at 3 54 20 PM" src="https://user-images.githubusercontent.com/89816272/193913103-35592d6b-5ed6-4664-aaf9-5a3b4433e494.png">

And here is the error that came up when I tried to check if library_interactive?.data.aspect_ratio_method === "MAX", and the typing for the library interactive data object.

<img width="704" alt="Screen Shot 2022-10-04 at 3 46 34 PM" src="https://user-images.githubusercontent.com/89816272/193913870-98cb0eaa-e9f4-40f4-84fe-ed607d3b765f.png">

<img width="319" alt="Screen Shot 2022-10-04 at 3 47 11 PM" src="https://user-images.githubusercontent.com/89816272/193913905-7ae27975-cf48-45c1-9327-ae71599b4db8.png">

Let me know what you think.